src/components/Header/_header.scss

```diff
@@ -0,0 +1,50 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+.#{$prefix}--header {
+  .#{$prefix}--skip-to-content:focus {
+    min-width: 240px;
+    justify-content: center;
+    border: $spacing-01 solid $ui-01;
+  }
+
+  &__menu-toggle {
+    display: block;
+    color: red;
```

Comment:
This should be a color token var. Is the red intended in the spec?

Reply:
that was a testing style and should not be included. will remove.